### PR TITLE
feat:-add navaigation component

### PR DESCRIPTION
**Pull request by shubhamkumar648**
navigation component
1.
nav2
Screenshot from 2022-02-11 19-17-50
ecom navbar
2.home page navbar

### CODE REVIEW

File: component/navigation.html [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+                <button class="button">
+                    <a href="#" class="link__nostyle">Home</a>
+                </button>
+
+                <button class="button">
+                    <a href="#" class="link__nostyle">Installment</a>
+                </button>
+
+                <button class="button">
+                    <a href="../pages/documentation.html" class="link__nostyle">Documnetation
```
Comment (ttahm3d) [inline component/navigation.html:28]: there is a typo.

File: styles/navigation.css [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+}
+
+.input_Serch_container {
+    flex-basis: 30%;
+}
+
+.input_serch {
+    width: 100%;
+    border-radius: 5px;
+    padding: 10px;
```
Comment (aatishpandey) [inline styles/navigation.css]: i think 1rem would work here

**Review by aatishpandey — COMMENTED** [review]
Rest all looks good

File: component/navigation.html [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+
+            <div class="navbarMain_body flex">
+
+                <!-- ecom header -->
+                <header class="ecom_header flex">
+                    <div class="brand_name">
+                        <a href="#" class="link__nostyle fs-xl brand_color">Grow</a>
+                    </div>
+                    <div class="input_Serch_container">
+                        <input type="serch" class="input_serch" placeholder="Serch">
```
Comment (tanishq20) [inline component/navigation.html]: typo: `search`

File: component/navigation.html (hunk 1 above)
Comment (therajatg) [inline component/navigation.html:28]: documentation spelling

File: styles/navigation.css (hunk 2 above)
Comment (shubhamkumar648) [reply styles/navigation.css]: yes thank you i will modify in next commit

File: component/navigation.html [hunk 4]
```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../styles/main.css" />
+    <title>GK UI</title>
+    <script src="https://kit.fontawesome.com/a4417b6562.js" crossorigin="anonymous"></script>
```
Comment (therajatg) [inline component/navigation.html:10]: script should come at end.

**Review by therajatg — APPROVED** [review]

**Review by tanishq20 — COMMENTED** [review]
Suggested small change.

**Review by aatishpandey — APPROVED** [review]
looks good to merge

**Review by ttahm3d — APPROVED** [review]
There is one typo apart from that everything looks good.

File: component/navigation.html (hunk 3 above)
Comment (shubhamkumar648) [reply component/navigation.html]: i will cahnge in next commit

File: component/navigation.html (hunk 4 above)
Comment (shubhamkumar648) [reply component/navigation.html:10]: ok  thank you , it will reflect on next commit

File: component/navigation.html (hunk 1 above)
Comment (shubhamkumar648) [reply component/navigation.html:28]: thank you tahir,it will reflect on next commit

**Pull request merged**